src/applications/mhv/secure-messaging/components/MessageThread/MessageThreadItem.jsx

```diff
@@ -23,104 +40,82 @@ const MessageThreadItem = props => {
     [props.printView],
   );
 
-  const handleExpand = e => {
-    // prevent messaage to expand/collapse when user is selecting text
-    if (window.getSelection().toString().length === 0) {
-      if (e.keyCode === 32) {
-        e.preventDefault(); // prevent from scrolling to the footer
-      }
-
-      // expanding/collapsing on Enter or Space key press for accessibility
-      if (
-        (e.key === 'Enter' || e.key === ' ' || e.type === 'click') &&
-        e.target.tagName !== 'a' // prevent from collapsing when user clicks on a link in the message
-      ) {
-        setIsExpanded(!isExpanded);
-      }
-
-      if (!isExpanded) {
-        dispatch(markMessageAsReadInThread(message.messageId));
-      }
+  const handleExpand = isPreloaded => {
+    if (!isPreloaded) {
+      dispatch(markMessageAsReadInThread(messageId, isDraftThread));
     }
+    setIsExpanded(!isExpanded);
   };
 
-  const hasAttachments = useMemo(
-    () => {
-      return (
-        message.attachment ||
-        message.hasAttachments ||
-        message.attachments?.length
-      );
-    },
-    [message.attachment, message.hasAttachments, message.attachments],
-  );
+  // const header = () => {
+  //   return (
+  //     <>
+  //       <i
+  //         className="unread-icon fas fa-circle"
+  //         aria-hidden
+  //         style={{ visibility: isRead === true ? 'hidden' : '' }}
+  //       />
+  //       {dateFormat(sentDate, 'MMMM D, YYYY [at] h:mm a z')}
+  //     </>
+  //   );
+  // };
 
-  const ariaLabel = useMemo(
-    () => {
-      return `${!isRead ? 'New' : ''} message ${
-        hasAttachments ? 'with attachment' : ''
-      } from ${message.senderName}, ${dateFormat(
-        message.sentDate,
-        'MMMM D, YYYY [at] h:mm a z',
-      )}. Click to ${isExpanded ? 'Collapse message' : 'Expand message'}`;
-    },
-    [hasAttachments, isExpanded, isRead, message],
-  );
+  // const subheader = () => {
+  //   return (
+  //     <>
+  //       <i
+  //         className="fas fa-paperclip"
+  //         aria-hidden
+  //         style={{ visibility: attachments !== true ? 'hidden' : '' }}
+  //       />
+  //       {from}
+  //     </>
+  //   );
+  // };

```

Comment:
Do we need to keep this?

Reply:
yes, this should stay until accordion is ready to accept nodes in the header and subheader